Statistics/Distribution/NegativeBinomial.hs

```diff
@@ -166,14 +166,14 @@ directEntropy d =
   dropWhile (>= -m_epsilon) $
   [ let x = probability d k in x * log x | k <- [0..]]
 
--- | Construct negative binomial distribution. Number of failures /r/
+-- | Construct negative binomial distribution. Number of successes /r/
```

Comment:
This got me really confused.

The "output" of the distribution is probability of number of failures before we get r-th success.
But here r is the (input) parameter of the distribution representing the number of  **successes** that need to occur overall.